src/test/java/net/imglib2/algorithm/morphology/MorphologyOpsTest.java

```diff
@@ -0,0 +1,180 @@
+package net.imglib2.algorithm.morphology;
+
+import ij.ImageJ;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+
+import net.imglib2.RandomAccess;
+import net.imglib2.algorithm.morphology.neighborhoods.CenteredRectangleShape;
+import net.imglib2.algorithm.region.localneighborhood.HyperSphereShape;
+import net.imglib2.algorithm.region.localneighborhood.RectangleShape;
+import net.imglib2.algorithm.region.localneighborhood.Shape;
+import net.imglib2.img.Img;
+import net.imglib2.img.array.ArrayImg;
+import net.imglib2.img.array.ArrayImgs;
+import net.imglib2.img.array.ArrayRandomAccess;
+import net.imglib2.img.basictypeaccess.array.ByteArray;
+import net.imglib2.img.display.imagej.ImageJFunctions;
+import net.imglib2.img.list.ListCursor;
+import net.imglib2.img.list.ListImg;
+import net.imglib2.img.list.ListImgFactory;
+import net.imglib2.type.Type;
+import net.imglib2.type.numeric.integer.UnsignedByteType;
+
+public class MorphologyOpsTest {
```

Comment:
We may want to be careful using the name `Ops` outside of the ImageJ OPS project. Could be confusing.